gke-scripts/10-apis.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+function enable_apis {
+  # Enable TD and other required APIs.
+  gcloud services enable \
+    container.googleapis.com \
```

Comment:
Does it make sense to add a script that creates a cluster too?

Reply:
So I purposely didn't include them for a couple of reasons:

- It is best to leave the infra creation stuff (such as GKE cluster) in the UG since it already has cluster creation instructions for the hello world example. The cluster creation step has nothing specific about the Wallet example and we are not providing complete automation here. No need to make these scripts unnecessarily dependent on GKE cluster creation commands, flags, versions etc which may change over time
- user may want to reuse an existing cluster

Almost the same argument for private CA resources. Providing automation for those makes a user's life easy but I don't see how it has anything to do with the Wallet example. It is best to provide that part in the UG

Reply:
I still think it's a good idea to put the scripts in there and talk about them in some way here. The main reason to have the scripts here is for completeness for people on our team who would want to quickly set things up and dont have to read through the public docs. And for people who directly get to our repo (and not to the public docs). The repo itself should in someway be able to guide someone landing there to setup everything. Wdyt?

Reply:
So I will add the cluster and private CA creation scripts here and add some docs for it. In the UG we could then reference them if needed

Reply:
Done.